src/bot/commands/delete.ts

```diff
@@ -0,0 +1,147 @@
+import Command from '../lib/command';
+import {
+  APIEmbed,
+  ApplicationCommandOptionType,
+  ApplicationCommandType,
+  bold,
+  ButtonStyle,
+  ChatInputCommandInteraction,
+  codeBlock,
+  ComponentType,
+  Message,
+  roleMention,
+  TextChannel
+} from 'discord.js';
+import {
+  buildJokeDisplay,
+  interactionInfo,
+  interactionProblem,
+  interactionValidate,
+  isEmbedable,
+  waitForInteraction
+} from '../utils';
+import { CategoriesRefs } from '../../typings';
+import { jokeById, jokeByQuestion } from '../../controllers';
+import { Colors, logsChannelId } from '../constants';
+import Jokes from '../../jokes';
+import prisma from '../../prisma';
+import { stripIndents } from 'common-tags';
+
+export default class DeleteCommand extends Command {
+  constructor() {
+    super({
+      name: 'delete',
+      description: 'Supprimer une blague',
+      type: ApplicationCommandType.ChatInput,
+      options: [
+        {
+          type: ApplicationCommandOptionType.String,
+          name: 'recherche',
```

Comment:
```suggestion
          name: 'query',
```
toujours en anglais, tu peux ajouter `nameLocalizations` si tu veux qu'il soit affiché en français mais dans le code ça doit être en anglais.
https://discord.js.org/#/docs/main/main/typedef/ApplicationCommandData

Reply:
Ouais, mais si on veut mettre les slashLocalization faudrais changer d'eux trois trucs dans le lib/command (sauf si j'ai loupé une pr) mais si c'est pas déjà fais, ça je pourais m'en occuper...